src/AspNet.Security.OAuth.Etsy/EtsyAuthenticationOptions.cs

```diff
@@ -0,0 +1,125 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+using Microsoft.Extensions.Options;
+using static AspNet.Security.OAuth.Etsy.EtsyAuthenticationConstants;
+
+namespace AspNet.Security.OAuth.Etsy;
+
+/// <summary>
+/// Defines a set of options used by <see cref="EtsyAuthenticationHandler"/>.
+/// </summary>
+public class EtsyAuthenticationOptions : OAuthOptions
+{
+    public bool IncludeDetailedUserInfo { get; set; }
+
+    public EtsyAuthenticationOptions()
+    {
+        ClaimsIssuer = EtsyAuthenticationDefaults.Issuer;
+        CallbackPath = EtsyAuthenticationDefaults.CallbackPath;
+
+        AuthorizationEndpoint = EtsyAuthenticationDefaults.AuthorizationEndpoint;
+        TokenEndpoint = EtsyAuthenticationDefaults.TokenEndpoint;
+        UserInformationEndpoint = EtsyAuthenticationDefaults.UserInformationEndpoint;
+
+        // Enable PKCE by default (required by Etsy)
+        UsePkce = true;
+
+        // Enable refresh token support
+        SaveTokens = true;
```

Comment:
These comments don't really add any value here.

```suggestion
        UsePkce = true;
        SaveTokens = true;
```